apps/api-wallet/src/app/utils/transactions/insertTransaction.util.ts

```diff
@@ -0,0 +1,31 @@
+import { DatabaseService } from '../../database/services/db_service';
+
+export class DbTransactions {
+  public static async insertTransaction(
+    databaseService: DatabaseService,
+    senderMobileNo: string,
+    receiverMobileNo: string,
+    amount: number,
+    serviceFee: number,
+    serviceType: string
+  ): Promise<any> {
+    const connection = await databaseService.startTransaction();
+    try {
+      const user = await databaseService.getQueryResult('insert_Transaction', [
+        senderMobileNo,
+        receiverMobileNo,
+        amount,
+        serviceFee,
+        serviceType,
+      ]);
+
+      await connection.commit();
```

Comment:
moving forward let's create a util function for this lang in database service so that we can reuse this lang

Reply:
ok bai..imove lang naku..